### PR TITLE
Fix/Avatar upload

### PR DESCRIPTION
**Pull request by jphinning**
### Changes
- Rotate picture added to upload modal
image

- Avatar remove and preview also added to profile picture
image

### CODE REVIEW

File: src/components/photoUpload/PhotoUpload.tsx [hunk 1]
```diff
@@ -1,35 +1,60 @@
...
 }
 
 function PhotoUpload({ avatarUrl, refetch }: IPhotoUploadProps) {
+  const { t } = useTranslation();
   const { authToken } = useAuth();
 
-  const [isLoading, setIsLoading] = useState<boolean>(false);
-  const [imageUrl, setImageUrl] = useState<string | null>(null);
-  const { t } = useTranslation();
+  const [previewOpen, setPreviewOpen] = useState(false);
```
Comment (milkavladislav) [inline src/components/photoUpload/PhotoUpload.tsx]: why you deleted types?

File: src/components/photoUpload/PhotoUpload.tsx [hunk 2]
```diff
@@ -39,48 +64,45 @@ function PhotoUpload({ avatarUrl, refetch }: IPhotoUploadProps) {
...
       return;
     }
 
-    getBase64(info.file.originFileObj as RcFile, async (url) => {
-      setIsLoading(false);
-      setImageUrl(url);
-      refetch();
-      toast.success(t("profileSuccessSubmitMessage"));
-    });
+    if (file.status === "done") toast.success(t("profileSuccessSubmitMessage"));
```
Comment (milkavladislav) [inline src/components/photoUpload/PhotoUpload.tsx]: IMHO status should be in consts

**Review by milkavladislav — APPROVED** [review]

File: src/components/photoUpload/PhotoUpload.tsx (hunk 1 above)
Comment (jphinning) [reply src/components/photoUpload/PhotoUpload.tsx]: Types added

File: src/components/photoUpload/PhotoUpload.tsx (hunk 2 above)
Comment (jphinning) [reply src/components/photoUpload/PhotoUpload.tsx]: Status consts added

File: src/components/photoUpload/PhotoUpload.tsx [hunk 3]
```diff
@@ -1,35 +1,60 @@
...
 
 function PhotoUpload({ avatarUrl, refetch }: IPhotoUploadProps) {
+  const { t } = useTranslation();
   const { authToken } = useAuth();
 
-  const [isLoading, setIsLoading] = useState<boolean>(false);
-  const [imageUrl, setImageUrl] = useState<string | null>(null);
-  const { t } = useTranslation();
+  const [previewOpen, setPreviewOpen] = useState<boolean>(false);
+  const [previewImage, setPreviewImage] = useState<string>("");
```
Comment (lyudad) [inline src/components/photoUpload/PhotoUpload.tsx]: can we use null?

Comment (jphinning) [reply src/components/photoUpload/PhotoUpload.tsx]: The img src accepts undefined, fixed

**Review by lyudad — APPROVED** [review]

**Pull request merged**